assets/css/_draft_picks.scss

```diff
@@ -46,3 +46,29 @@
     }
   }
 }
+
+.draft-summary-table {
+  margin-bottom: 2em;
+  
```

Comment:
Line contains trailing whitespace

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#trailingwhitespace)